source/postcard-rpc/src/standard_icd.rs

```diff
@@ -57,6 +57,22 @@ pub enum WireError {
     KeyTooSmall,
 }
 
+impl core::fmt::Display for WireError {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        match self {
+            WireError::FrameTooLong(e) => write!(f, "The frame exceeded the buffering capabilities of the server: {} > {}", e.len, e.max),
+            WireError::FrameTooShort(e) => write!(f, "The frame was shorter than the minimum frame size and was rejected: {}", e.len),
+            WireError::DeserFailed => write!(f, "Deserialization of a message failed"),
```

Comment:
For the ones that don't actually format anything, I think you can just call `f.push_str()` which likely is more efficient.